frontend/src/features/admin-form/responses/IndividualResponsePage/IndividualPaymentResponse.tsx

```diff
@@ -0,0 +1,76 @@
+import { Stack, Text } from '@chakra-ui/react'
+
+import { useIndividualPaymentSubmission } from './queries'
+
+export const IndividualPaymentResponse = (submissionId: {
+  submissionId: string
+}): JSX.Element => {
+  if (!submissionId) throw new Error('Missing submissionId')
+
+  const {
+    data: paymentData,
+    isLoading: isPaymentLoading,
+    isError: isPaymentError,
+  } = useIndividualPaymentSubmission()
+
+  return (
+    <Stack>
+      <Text
+        textStyle="h2"
+        as="h2"
+        color="primary.500"
+        mb="0.5rem"
+        _notFirst={{ mt: '2.5rem' }}
+      >
+        Payment
+      </Text>
+      {isPaymentLoading || isPaymentError ? (
```

Comment:
from @wanlingt, looking forward the loading & error portion will likely be covered under useIndividualSubmission() fin IndividualResponsePage.

we can potentially leverage on that, and leverage the loading decryption skeleton from Individual Response page instead of returning `<Text>Payment was not enabled when this form was submitted</Text>` (Which isn't being called during error anyway).